Documentation/ProjectFile/prj/processes/process/RichardsComponentTransport/t_specific_body_force.md

```diff
@@ -0,0 +1,3 @@
+Specific body forces applied to fluid.
+
+It is usually used to apply gravitational forces. Given as a 2d or 3d vector.
```

Comment:
Nit picking: it's probably copy-paste, but I'd think it also applies to 1d problems...